.github/workflows/workflow_canceller.yml

```diff
@@ -0,0 +1,19 @@
+name: Automatic Workflow Canceller
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      # Push events on develop branch
+      - develop
+
+jobs:
+  cancel:
+    name: Cancel Previous Runs
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: styfle/cancel-workflow-action@0.6.0
```

Comment:
Should we mention the reference? Might be a learning curve
https://github.com/styfle/cancel-workflow-action

Reply:
Sure. I sort of expect people to be able to find these actions directly, but the additional comment doesn't hurt.